pkg/scheduler/extender.go

```diff
@@ -254,7 +255,8 @@ func convertToNodeNameToMetaVictims(
 	nodeNameToMetaVictims := map[string]*extenderv1.MetaVictims{}
 	for node, victims := range nodeNameToVictims {
 		metaVictims := &extenderv1.MetaVictims{
-			Pods: []*extenderv1.MetaPod{},
+			Pods:             []*extenderv1.MetaPod{},
```

Comment:
the function name sounds strange, can you pls change it to just `convertToMetaVictims`